pkg/pods/pod.go

```diff
@@ -27,20 +27,19 @@ func PodPath(manifestId string) string {
 }
 
 type Pod struct {
+	Id             string
```

Comment:
This seemed like the lesser of the evils between extracting Id from the path or from the manifest